lib/wpcom-error-handler/wpcom-error-handler.php

```diff
@@ -24,6 +24,13 @@ function wpcom_error_shutdown() {
 	}
 }
 
+/**
+ * @param int $type
+ * @param string $message
+ * @param string $file
+ * @param int $line
+ * @return void
```

Comment:
Adding alignment and descriptions for each arg would be useful here, and on other instances where arg types have been added to the DocBlock.

Reply:
Fixed in 989b3dc93